examples/darwin-framework-tool/commands/common/CHIPCommandBridge.mm

```diff
@@ -237,12 +261,21 @@
             params.nodeId = @(mCommissionerNodeId.Value());
         }
 
-        // We're not sure whether we're creating a new fabric or using an existing one, so just try both.
-        auto controller = [factory createControllerOnExistingFabric:params error:&error];
-        if (controller == nil) {
-            // Maybe we didn't have this fabric yet.
-            params.vendorID = @(mCommissionerVendorId.ValueOr(chip::VendorId::TestVendor1));
-            controller = [factory createControllerOnNewFabric:params error:&error];
+        MTRDeviceController * controller = nil;
+        if (mUseXPC.ValueOr(false) || mUseXPCServiceName.HasValue()) {
```

Comment:
I don't think the XPC stuff makes sense in the shared-storage situation.  Why are we trying to support that?  The framework doesn't really envision XPC being used with shared storage.

Reply:
I included this for a few reasons:
 - It may help with debugging issues related to the “old” setup.
 - Since we still support the shared storage scenario, it’s more consistent to maintain feature parity.
 - I already had this code locally, and I’ve structured the PR so that removing it is straightforward—just delete the file and remove two lines in CHIPCommandBridge. This makes it easy to remove later if needed.